app/Resources/FOSUserBundle/views/Security/login.html.twig

```diff
@@ -1,41 +1,51 @@
 
 {% extends 'base.html.twig' %}
 
+{% block body %}
 
+    <div id="login-form" class="form">
 
-{% block body %}
+    {% if is_granted('ROLE_USER') %}
 
+      <p>{{ 'security.login.already_logged_in'|trans }}</p>
 
-  <div id="login-form" class="form">
+    {% else %}
 
-      
-<form action="{{ path("fos_user_security_check") }}" method="post">
-    {% if csrf_token %}
-        <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
-    {% endif %}
+      <form action="{{ path("fos_user_security_check") }}" method="post">
 
+      {% if csrf_token %}
+        <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
+      {% endif %}
 
         <h1>{{ 'security.login.welcome'|trans }}</h1>
         <div class="form-group">
-        <label for="username">{{ 'security.login.username'|trans }}</label>
-        <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" />
+          <label for="username">{{ 'security.login.username'|trans }}</label>
+          <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" />
         </div>
-         
+
         <div class="form-group">
-        <label for="password">{{ 'security.login.password'|trans }}</label>
-        <input type="password" id="password" name="_password" required="required" />
+          <label for="password">{{ 'security.login.password'|trans }}</label>
+          <input type="password" id="password" name="_password" required="required" />
         </div>
         <div>
-        <input type="checkbox" id="remember_me" name="_remember_me" value="on" />
-        <label for="remember_me">{{ 'security.login.remember_me'|trans }}</label>
+          <input type="checkbox" id="remember_me" name="_remember_me" value="on" />
+          <label for="remember_me">{{ 'security.login.remember_me'|trans }}</label>
         </div>
         <input type="hidden" name="_target_path" value="/dashboard" />
         <button type="submit" id="_submit" name="_submit">{{ 'security.login.submit'|trans }}</button>
 
-</form>
-        
-{% if error %}
-    <div class="form-group">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
-{% endif %}
-  </div>
-{% endblock %}
\ No newline at end of file
+        <div class="form-group">
+          <a href={{ path('fos_user_resetting_request') }}>{{ 'security.login.forgotten_password'|trans }}</a>
```

Comment:
Please add some style to the link, it is merely invisible right now ! You can use the ```button``` class on the ```a``` tag to get a basic style. The additional class ```warning``` can also be set to get an orange button if you like.

Reply:
I already tried with button warning, but as I said on Discord, it's too much. But I haven't even thought about button, simply.